### ISSUE TITLE
Ubuntu/Linux - II want to add a new folder location using the command line

### ISSUE THREAD

**Issue by scisys-bonsor-s** [aegif/CmisSync#548]
Users may only have xterm access to a remote system, so how can they setup cmissync via the commandline - rather than edit the config.xml

**Comment by nicolas-raoul** [comment]
Interesting idea!
How about editing config.xml via vi or a similar tool?

**Comment by scisys-bonsor-s** [comment]
What are the basic items I need to provide?
Repo location (http....)
Password (which should then get some encryption)
Limit remote folder to "My Files" only on Alfresco
Local folder view of files should be under a CmisSync folder, no multiple level directory from Main repository

This could be done via simple bash script (apart from the password?)

**Comment by nicolas-raoul** [comment]
You are right, providing a bash script would be a convenient way to avoid configuration mistakes.
Would you be able to write this bash script as ope source, maybe? :-)
A sample config can be generated using Windows or Mac.
The password algorithm can be found at https://github.com/aegif/CmisSync/blob/master/CmisSync.Auth/Crypto.cs#L123 I am sure it can be implemented in bash using an AES tool.
Cheers!

**Comment by scisys-bonsor-s** [comment]
OK.
Should be able to do something with
openssl enc -d -a -aes256 and base64,, but I'm having issues with the GetCryptoKey - maybe it's the odd character in it. Can I use that as the decryption password, or do I need to remove base64 first before pushing it through openssl ?

Password - >  base64 encode -> openssl enc to encode ?
